File: maria/php/read/maria_read.php
#! /usr/bin/php
<?php
/*
	maria_read.php

					Jun/29/2017

*/
// --------------------------------------------------------------------
$path="../../../common/php_common";
set_include_path (get_include_path() . PATH_SEPARATOR . $path);
include "sql_display.php";
// --------------------------------------------------------------------
$host = 'localhost';
$user = 'scott';
$password = 'tiger123';
$db = 'city';

$dsn = 'mysql:dbname=' . $db . ';host=' . $host . ';charset=utf8';

fputs (STDERR,"*** 開始 ***\n");

try
	{
	$dbcon = new PDO ($dsn, $user,$password);
	disp_lower_proc ($dbcon);
	$dbcon = null;
	}
catch  (PDOException $e)
	{
	print('Error:'.$e->getMessage());
	}

fputs (STDERR,"*** 終了 ***\n");
// --------------------------------------------------------------------
?>	

File: maria/lua/read/maria_read.lua
#! /usr/bin/lua
-- ----------------------------------------------------
--	maria_read.lua
--
--					Nov/17/2014
-- ----------------------------------------------------
require "luasql.mysql"
require "sql_manipulate"
-- ----------------------------------------------------
print ("*** 開始 ***")

local dbname = "city"
local dbhost = "localhost"
local dbuser = "scott"
local dbpass = "tiger"

local env = assert(luasql.mysql())
local con = assert(env:connect(dbname, dbuser, dbpass, dbhost))

sql_display_proc (con)
con:close()
env:close()

print ("*** 終了 ***")

-- ----------------------------------------------------
